src/App.tsx

```diff
@@ -67,8 +68,9 @@ const App = () => {
         options={{ label: "Games" }}
       ></Resource>
       <Resource
-        name="users_view"
+        name="users"
```

Comment:
This probably broke the List view. You will probably need to override the resource for either the List or the Edit view.